app/src/main/java/com/sunlightlabs/android/congress/fragments/LegislatorListFragment.java

```diff
@@ -293,12 +293,16 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
 		public String nameFor(Legislator legislator) {
 			if (context.type == SEARCH_COMMITTEE) {
-				return legislator.title + ". " + legislator.firstName() + " " + legislator.last_name;
+				return legislator.title + ". " + legislator.first_name + " " + legislator.last_name;
 			} else
-				return legislator.last_name + ", " + legislator.firstName();
+				return legislator.last_name + ", " + legislator.first_name;
 		}
 
 		public String positionFor(Legislator legislator) {
+			// For cosponsor listings
+            if (legislator.state == null || legislator.party == null || legislator.district == null)
+			    return "";
```

Comment:
For cosponsors, we get very limited information, and so we just show their name and that's it. Feature request to restore this information here: https://github.com/propublica/congress-api-docs/issues/69